content/swift/concepts/inheritance/inheritance.md

```diff
@@ -0,0 +1,135 @@
+---
+Title: 'Inheritance'
+Description: 'Inheritance allows for reusing of code by sharing features between classes.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Closures'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Inheritance allows for the transferring of properties, methods, and other characteristics between classes. This is a feature that differentiates Swift classes from other types.
```

Comment:
```suggestion
Inheritance allows properties, methods, and other characteristics to be transferred between classes. This is a feature that differentiates Swift classes from other types.
```

Did we want to say "transferred" or "shared"?

Maybe we can also link out to Swift data types when we mention "types."

Reply:
yeah, I was also debating on whether to say 'shared' or 'transferred' and then just ended up saying transferred since the parent class gives to the child but the parent class can't use the child properties and methods.

Reply:
I'll add the link